pkg/appfile/appfile.go

```diff
@@ -21,6 +21,7 @@ var (
 )
 
 const DefaultAppfilePath = "./vela.yaml"
+const DefaultVersion = "1.0-alpha.1"
```

Comment:
for a version of format, we usually name it like  `v1alpha1` (k8s) or `2010-09-09`(cloud formation)

Reply:
v1alpha1 is k8s specific and it is not even semver.
See https://docs.docker.com/compose/:

```
version: "3.8"
services:
```

Reply:
yes, something like `3.8` is also good for me.

Reply:
Using `v1alpha1` will complicate developers who are not familiar with k8s

Reply:
what about just `v1`?

Reply:
It is unwise to declare v1 for now...
This is not something we want to solve before kubecon. I would suggest let's just follow compose spec for now.

Reply:
Since it is alpha we can change that later

Reply:
I removed the `version` field from Appfile. 
This is not something deserved a place right now